settings.gradle.kts

```diff
@@ -27,12 +27,12 @@ rootProject.apply {
    include(":widgets")
    project(":widgets").apply {
       name = "widgets"
-      projectDir = file("app")
+      projectDir = file("app/widgets")
 
-      file("app/widgets").listFiles().forEach {
+      projectDir.listFiles().forEach {
          include(":widgets:${it.name}")
          project(":widgets:${it.name}").apply {
-            name = "player-widgets-${it.name}"
+            name = "widget-${it.name}"
```

Comment:
The name was deliberate so the modules' alphabetical order mirrors the module structure at all times. But I'll accept this and see what happens.

Reply:
Why is this important?
For me the module names were simply far too long, especially in the Project Hierarchy of IDEA they were cut off before I could read anything of interest.

Reply:
Because it is logical and intuitive.

Reply:
but where do you see all modules listed?

Reply:
In IDE, but I see the point.